train_gpt2.cu

```diff
@@ -1546,6 +1552,28 @@ int main(int argc, char *argv[]) {
     Tokenizer tokenizer;
     tokenizer_init(&tokenizer, "gpt2_tokenizer.bin");
 
+    // set up learning rate scheduler
```

Comment:
we could move this new block inside the schedulers file. and the block below as well.

Reply:
Done, simplified the code, it's ready.